test/docfx.Test/common/TestUtility.cs

```diff
@@ -104,7 +104,10 @@ void Walk(HtmlNode node, int level)
                         sb.Append(node.Name);
                         foreach (var attr in node.Attributes.OrderBy(a => a.Name))
                         {
-                            sb.Append($" {attr.Name}=\"{TrimWhiteSpace(attr.Value)}\"");
+                            if (attr.Name != "data-linktype")
```

Comment:
why not apply for `data-linktype`?

Reply:
I not exclude here, that means modifying a lot of test cases, because this is not behind a legacy flag now, and every link will have data-linktype attribute